src/Processors/Transforms/WindowTransform.cpp

```diff
@@ -2480,7 +2480,7 @@ struct WindowFunctionNthValue final : public WindowFunction
         if (offset <= 0)
         {
             throw Exception(ErrorCodes::BAD_ARGUMENTS,
-                "The offset for function {} must be in (0, {}], {} given",
+                "The offset for function {} must be in (1, {}], {} given",
```

Comment:
Seems unrelated

Reply:
It is indeed unrelated. Is something I found along the way and I piggybacked it on this PR. Here's the commit: https://github.com/ClickHouse/ClickHouse/pull/66532/commits/0557ef17304eaab590485c4d4038b7eef61ce920
Should I open a separate PR just for this?